src/analysis/liveness-transfer-function.h

```diff
@@ -0,0 +1,101 @@
+#ifndef wasm_analysis_liveness_transfer_function_h
+#define wasm_analysis_liveness_transfer_function_h
+
+#include "lattice.h"
+#include "monotone-analyzer.h"
+
+namespace wasm::analysis {
+
+class LivenessTransferFunction : public Visitor<LivenessTransferFunction> {
+  FinitePowersetLattice::Element* currState;
+
+public:
+  LivenessTransferFunction() : currState(nullptr) {}
+
+  // Transfer function implementation. Modifies the state for a particular
+  // expression type. In our current limited implementation, we just update
+  // state on gets and sets of local indices.
```

Comment:
Is there anything more in a non-limited future version? I think that's all there is to liveness. So I would shorten this comment. I'd also add a brief reminder of how liveness works. So perhaps the whole comment could be:
```
Transfer function implementation. A local becomes live before a
get, and dead before a set.
```
(that also reminds the reader of the reverse ordering that is used below)

Reply:
I was not sure if more logic was needed to account for copies as was done in the `LivenessWalker` . Also whether we wanted to evaluate liveness effects for other expressions. However, for the basic liveness functionality, you're right, that's all there is. I have changed the comment according to your suggestion.